controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -94,8 +94,12 @@ func configureServiceMeshFeatures() feature.FeaturesProvider {
 			return cfgMapErr
 		}
 
+		authNs := servicemesh.CreateAuthNamespace(serviceMeshSpec.Auth.Namespace,
+			handler.DSCInitializationSpec.ApplicationsNamespace)
+
 		extAuthzErr := feature.CreateFeature("mesh-control-plane-external-authz").
 			For(handler).
+			TargetNamespace(authNs).
```

Comment:
I think a change in `mesh-authz-ext-provider.patch.tmpl` is missing:
https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/pkg/feature/templates/servicemesh/authorino/mesh-authz-ext-provider.patch.tmpl#L12